prepare/download.sh

```
@@ -0,0 +1,563 @@
+setup_ghcr_auth() {
+    local username="${GHCR_USERNAME:-xiilab}"
+    local token="$GHCR_TOKEN"
+
+    echo -e "${BLUE}🔓 GHCR 인증 파일 생성 중...${NC}"
+    echo "   Registry: ghcr.io"
+    echo "   Username: $username"
+    echo ""
+
+    # auth.json 파일 생성 (Container Runtime이 자동으로 사용)
+    local auth_file="${SCRIPT_DIR}/.ghcr-auth.json"
+    local auth_base64=$(echo -n "$username:$token" | base64)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)